tests/Asset/AssetPackageTest.php

```
@@ -107,6 +121,81 @@ public function testAddNewDependencies(): void
+        self::assertSame(
+            'file:./fresh/path',
+            $assetPackage->getPackage()['dependencies']['@composer-asset/foo--bar'],
+        );
+    }
+
+    public function testDependencyPathUsesInjectedFilesystemForPortableRelativePath(): void
+    {
+        $path = 'asset/foo/package.json';
+        $consumerPath = '/consumer/package.json';
+        $relativePath = '..\\asset\\foo';
+        $fs = $this->createMock(ComposerFilesystem::class);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $fs. Configured minimum length is 3. (undefined)

(ShortVariable)